### PR TITLE
Allow clustering on \ as well as : and fix newlines

### PR DESCRIPTION
**Pull request by rjmunro**
Some fixes I had to make to get this to work with my setup.

### CODE REVIEW

File: Graphviz/DoctrineMetadataGraph.php [hunk 1]
```diff
@@ -85,11 +85,11 @@ private function getEntityLabel($class, $entity)
         $result = '{{<__class__> '.$class.'|';
 
         foreach ($entity['associations'] as $name => $val) {
-            $result .= '<'.$name.'> '.$name.' : '.$val.'\l|';
+            $result .= '<'.$name.'> '.$name.' : '.$val."\n|";
```
Comment (OskarStark) [inline Graphviz/DoctrineMetadataGraph.php]: maybe use PHP_EOL?

Comment (rjmunro) [inline Graphviz/DoctrineMetadataGraph.php]: Yeah, that would be better.

**Comment by jverdeyen** [comment]
:+1: Can you merge this?

**Comment by mvhirsch** [comment]
:+1: 

this will fix #2, #3 and #10

**Comment by mvhirsch** [comment]
@alexandresalome please merge

**Comment by OskarStark** [comment]
worked for me too, please merge this PR  @alexandresalome 

:+1:

**Pull request merged**